src/jsonify/parse.ts

```diff
@@ -30,9 +30,13 @@ function unpack(
   custom: CustomParser | undefined,
 ): void {
   if (idx in hydrated) return;
-
+  
   const current = arr[idx];
   if (typeof current === "number") {
+    if(idx !== 0) {
```

Comment:
Running `deno fmt` should resolve the formatting 👍

Reply:
Thanks for the comment, I'll do it 👍